README.md

```diff
@@ -196,56 +173,60 @@ public class DroidModule {
 
 ##### Inversion of import
 
-Sometimes you may want to specify that a module should be imported by another modules and/or components without
+Sometimes you may want to specify that a module should be imported by another modules without
 modifying them. It can be achieved by applying the `@ImportedBy` annotation to the module that needs to be imported:
 
 ```java
-@Component
-public class DroidComponent {
-  /* ... */
+
+@Module
+public class RobotModule {
+    /* ... */
 }
 
 @Module
-@ImportedBy(DroidComponent.class)
+@ImportedBy(RobotModule.class)
 public class DroidModule {
-  /* ... */
+    /* ... */
 }
 ```
 
 ##### Contracts
 
-While components allow to create an `Injector` that provides an instance by its type, there's also a way to create a
-statically typed dependency provider, which is a contract. A contract is an interface that may contain any number of
-methods and can extend other interfaces. Contract's methods must have no arguments and must return a non-`void` type.
+Contract is the statically typed dependency provider. A contract is an interface that may contain
+any number of methods and can extend other interfaces. Contract's methods must have no arguments and
```

Comment:
Suggestion:

> A contract is an interface that may contain
any number of methods and can extend other interfaces which act as a typed dependency provider.